Execution fails if using WSL and username contains spaces

## Describe the Bug

The execution currently fails if using WSL and the username contains spaces. This is due to improper escaping of space characters in GitExe and CygPathExe.

## Steps to Reproduce

1. Create a Windows username with spaces on the name, so its .nuget folder is in a path that contains spaces, such as `C:\Users\Example User\.nuget\packages\gitinfo\2.2.0\build`.
2. Attempt to run a project that has GitInfo setup.

On the output console the following will be displayed:
```
4>"C:\Users\Example" is not recognized as an internal or external command,
4>operable program or batch file.
4>C:\Users\Example User\.nuget\packages\gitinfo\2.2.0\build\GitInfo.targets(1088,3): warning MSB3073: Command "..." exited with error code 9009.
4>"--version" is not recognized as an internal or external command,
4>operable program or batch file.
4>C:\Users\Example User\.nuget\packages\gitinfo\2.2.0\build\GitInfo.targets(203,3): warning MSB3073: Command "..." exited with error code 9009.
4>C:\Users\Example User\.nuget\packages\gitinfo\2.2.0\build\GitInfo.targets(212,3): error : Failed to run `git --version`. Git may not be properly installed: 
4>C:\Users\Example User\.nuget\packages\gitinfo\2.2.0\build\GitInfo.targets(212,3): error :  "--version" is not recognized as an internal or external command,;operable program or batch file.
```

## Expected Behavior

GitInfo should run just fine.

## Version Info

Present in latest 2.2.0

--- a/src/GitInfo/build/GitInfo.targets
+++ b/src/GitInfo/build/GitInfo.targets
@@ -239,7 +239,7 @@
     </PropertyGroup>
 
     <!-- Account for cygwin/WSL separately -->
-    <Exec Command='"$(CygPathExe)" -w "$(_GitOutput)"'
+    <Exec Command='$(CygPathExe) -w "$(_GitOutput)"'
 			  EchoOff='true'
 			  WorkingDirectory="$(GitInfoBaseDir)"
 			  StandardErrorImportance='high'
@@ -297,7 +297,7 @@
     </Exec>
 
     <!-- Account for cygwin/WSL separately -->
-    <Exec Command='"$(CygPathExe)" -w "$(_GitCommonDir)"'
+    <Exec Command='$(CygPathExe) -w "$(_GitCommonDir)"'
 			  EchoOff='true'
 			  WorkingDirectory="$(GitInfoBaseDir)"
 			  StandardErrorImportance='high'
@@ -588,7 +588,7 @@
     <Message Text="Using base version from version file $(GitVersionFile)'" Importance="normal" />
 
     <!-- Account for cygwin/WSL separately -->
-    <Exec Command='"$(CygPathExe)" -u "$(_GitVersionFile)"'
+    <Exec Command='$(CygPathExe) -u "$(_GitVersionFile)"'
 			  EchoOff='true'
 			  WorkingDirectory="$(GitRoot)"
 			  StandardErrorImportance='high'
@@ -630,7 +630,7 @@
     </PropertyGroup>
 
     <!-- Account for cygwin/WSL separately -->
-    <Exec Command='"$(CygPathExe)" -u $(_GitCommitsRelativeTo)'
+    <Exec Command='$(CygPathExe) -u $(_GitCommitsRelativeTo)'
 			  EchoOff='true'
 			  WorkingDirectory="$(GitRoot)"
 			  StandardErrorImportance='high'
@@ -1092,7 +1092,7 @@
 
     <!-- If we didn't find it in the PATH nor the above locations, check for git installed in WSL -->
     <Exec Condition="'$(GitExe)' == ''"
-			  Command="$(MSBuildThisFileDirectory)wslrun.cmd git --version"
+			  Command='"$(MSBuildThisFileDirectory)wslrun.cmd" git --version'
 			  EchoOff='true'
 			  ContinueOnError='true'
 			  StandardErrorImportance='high'
@@ -1101,8 +1101,8 @@
       <Output TaskParameter="ExitCode" PropertyName="MSBuildLastExitCode" />
     </Exec>
     <PropertyGroup Condition="'$(GitExe)' == '' And '$(MSBuildLastExitCode)' == '0'">
-      <GitExe>$(MSBuildThisFileDirectory)wslrun.cmd git</GitExe>
-      <CygPathExe>$(MSBuildThisFileDirectory)wslpath.cmd</CygPathExe>
+      <GitExe>"$(MSBuildThisFileDirectory)wslrun.cmd" git</GitExe>
+      <CygPathExe>"$(MSBuildThisFileDirectory)wslpath.cmd"</CygPathExe>
     </PropertyGroup>
 
     <PropertyGroup Condition="'$(GitExe)' == ''">

--- a/src/GitInfo/build/wslpath.cmd
+++ b/src/GitInfo/build/wslpath.cmd
@@ -17,10 +17,10 @@ if "%1" == "-u" shift /1
 REM Convert path to Linux
 if exist "%1\*" (pushd %1) else (pushd %~dp1)
 if ERRORLEVEL 1 goto :eof
-%WSLRUN% pwd
+"%WSLRUN%" pwd
 popd
 goto :eof
 
 :towindows
 REM Convert path to Windows
-%WSLRUN% cd "'%2'" ^&^& cmd.exe /c cd
+"%WSLRUN%" cd "'%2'" ^&^& cmd.exe /c cd

